src/assets/scss/forms.scss

```diff
@@ -361,10 +301,6 @@ ul.selected-options {
     outline-offset: 3px;
 }
 
-.c-checkbox input[type="checkbox"]:focus ~ .c-checkbox__icon {
-    outline: 3px solid transparent;
-}
```

Comment:
There is a difference with the reserved line 360 line, which is 2px and here is 3px, but it is not a big problem.

Reply:
Seemed like 2px was being applied the whole time so removed the 3px one